include/IECore/MurmurHash.h

```diff
@@ -84,6 +87,7 @@ class IECORE_API MurmurHash
 		inline bool operator < ( const MurmurHash &other ) const;
 
 		std::string toString() const;
+		void fromString( const std::string &repr );
```

Comment:
I really like the symmetry of these two functions - I think it makes it very clear that we're serialising to a string and deserialising back.